bot/plotting/random_plot_generator.py

```diff
@@ -95,37 +96,55 @@ def random_plot_generator(
                     "SEI": sei,
                 })
 
-            models = [
-                pybamm.lithium_ion.DFN(
-                    options=options
-                ),
-                pybamm.lithium_ion.SPM(
-                    options=options
-                ),
-                pybamm.lithium_ion.SPMe(
-                    options=options
-                ),
+            solvers = [
+                pybamm.CasadiSolver(mode="safe"),
+                pybamm.CasadiSolver(mode="fast with events")
             ]
 
-            model = random.choice(models)
+            solver = random.choice(solvers)
+
+            choice = random.randint(0, 3)
 
-            choice = random.randint(0, 2)
             if testing is True and provided_choice is not None:
                 choice = provided_choice
 
-            solvers = [
-                pybamm.CasadiSolver(mode="safe"),
-                pybamm.CasadiSolver(mode="fast"),
-                pybamm.CasadiSolver(mode="fast with events")
-            ]
+            if choice == 1:
```

Comment:
For example, if the `choice` is to choose between passing options or default, I think it should use if you always pass options and the choice is between passing an option dictionary or `None` (which should fall back to the default).

Reply:
Oh yes, I've made the changes. Thanks!